src/plugins/links/public/editor/open_editor_flyout.tsx

```diff
@@ -135,10 +139,12 @@ export async function openEditorFlyout(
         { theme: coreServices.theme, i18n: coreServices.i18n }
       ),
       {
+        tabIndex: 1,
```

Comment:
@Heenawter Which flyout is this `tabIndex` being assigned to? I wasn't able to find it in the DOM using a quick `document.querySelectorAll('[tabindex="1"]')` but had some unexpected tabbing behavior with Firefox + VO. It's entirely possible the new MacOS or VO has disrupted that flow; Firefox seemed okay by itself.

If there is a positive tabindex being applied, that could disrupt the natural tabbing order, so I'd like to evaluate that separately if it does appear in a place I didn't find.

Reply:
Oh, good catch - I think I accidentally left this in when I was testing different things for refocusing the main flyout after editing/adding links ; I've [removed it](https://github.com/elastic/kibana/pull/174772/commits/11468acc478b7ce8ec3a8bc36f0925d29aa27a6e#diff-7fabb771b837315761e4cd574c6f0fc636d02f5a38e47972a4b093252bdbc018L142), and it doesn't seem to impact the focusing behaviour 🙇